src/devinterp/slt/llc.py

```diff
@@ -115,19 +129,19 @@ def sample(self):
             "llc/means": self.llc_means.cpu().numpy(),
```

Comment:
i think i also just noticed that this should probably be `llc/mean` and not `llc/means` to be in line with the not online llc estimator above? also same w/ `llc/stds -> llc/std`